### ISSUE TITLE
MacOS Build Errors

### ISSUE THREAD

**Issue by BushTrain460615** [wxWidgets/wxWidgets#22779]
**Describe the problem**
Describe how exactly do you build wxWidgets, including the full `configure`
command line and/or `make` command line if relevant.

Please attach the full build log, but feel free to quote the relevant parts of
it here.

When using `configure`, please also attach `config.log` file.
[config.log](https://github.com/wxWidgets/wxWidgets/files/9498367/config.log)

**Platform and version information**
 - wxWidgets version you use: [git]
 - wxWidgets port you use: [wxOSX]
 - OS and its version: [macOS]

Build Log 

[errors.txt](https://github.com/wxWidgets/wxWidgets/files/9499049/errors.txt)

**Comment by oneeyeman1** [comment]
@BushTrain460615 ,
Are you trying to build the "git master"? If so - did you follow the instructions at https://wxwidgets.org/develop/code-repository/?

Also - is the errors are producing when building using configure or using CMake?

**Comment by BushTrain460615** [comment]
> @BushTrain460615 , Are you trying to build the "git master"? If so - did you follow the instructions at https://wxwidgets.org/develop/code-repository/?
> 
> Also - is the errors are producing when building using configure or using CMake?

It happens when I build with cmake

**Comment by vadz** [comment]
Your real problem is that you haven't checked out submodules, as both the build instructions and configure told you to do. So, let me try again: you need to use `git submodule update --init` if you hadn't cloned with them.

There are 2 other things:

1. Please don't attach output in RTF format (how do you even do this?) but just copy and paste plain text.
2. It doesn't make sense to use _both_ configure and CMake. You should either one or the other, but not both.

**Comment by BushTrain460615** [comment]
> 1. Please don't attach output in RTF format (how do you even do this?) but just copy and paste plain text.
I used the Text edit app on Mac that's why
